File: training-data/php/bubble_sort.php
<?php

function bubbleSort(array $items): array
{
    $arr = $items;
    $n = count($arr);
    for ($i = 0; $i < $n; $i++) {
        $swapped = false;
        for ($j = 0; $j < $n - $i - 1; $j++) {
            if ($arr[$j] > $arr[$j + 1]) {
                [$arr[$j], $arr[$j + 1]] = [$arr[$j + 1], $arr[$j]];
                $swapped = true;
            }
        }
        if (!$swapped) {
            break;
        }
    }
    return $arr;
}

print_r(bubbleSort([5, 3, 8, 1, 9, 2]));

File: training-data/julia/bubble_sort.jl
function bubble_sort(arr::Vector{Int})
    a = copy(arr)
    n = length(a)
    for i in 1:n
        for j in 1:(n - i)
            if a[j] > a[j + 1]
                a[j], a[j + 1] = a[j + 1], a[j]
            end
        end
    end
    return a
end

println(bubble_sort([5, 2, 9, 1, 5, 6]))
